lib/seeing_is_believing/evaluate_by_moving_files.rb

```diff
@@ -83,68 +87,72 @@ def set_back_to_initial_conditions
     end
 
     def write_program_to_file
-      File.open(filename, 'w') { |f| f.write program.to_s }
+      File.open(filename, 'w', external_encoding: "utf-8") { |f| f.write program.to_s }
```

Comment:
Here again, see the blog post for general background, and the comment above for specific commentary on why it may be effective but not correct.